### PR TITLE
[14.0][FIX] fieldservice: fix intermittent test failures

### PR DESCRIPTION
**Pull request by PicchiSeba**
Sometimes when submitting a PR automatic tests fail. I suspect it is caused by the evaluation of `datetime.now()` not being done immediately (it could be a machine with not enough resources available).

I am aware that this is more a 'hack' than i fix, but for the moment I am not aware of more elegant solutions.

EDIT: grammar, formatting

### CODE REVIEW

**Review by max3903 — APPROVED** [review]

**Review by hparfr — CHANGES_REQUESTED** [review]
Freezegun maybe ? 
https://github.com/odoo/odoo/blob/14.0/addons/project/tests/test_project_recurrence.py#L10

**Comment by PicchiSeba** [comment]
I applied the suggestion

File: fieldservice/tests/test_fsm_order.py [hunk 1]
```diff
@@ -20,50 +22,53 @@ def test_fsm_order(self):
...
+                order.priority = priority
+                order.request_late = False
+                vals = {"request_early": fields.Datetime.today(), "priority": priority}
+                vals = order._compute_request_late(vals)
+                self.assertEqual(
+                    vals["request_late"],
+                    order.request_early + timedelta(days=late_days),
+                )
+            # Test scheduled_date_start is not automatically set
+            # timedelta is necessary when the evaluation is not immediate
```
Comment (hparfr) [inline fieldservice/tests/test_fsm_order.py]: wrong comment

Comment (PicchiSeba) [reply fieldservice/tests/test_fsm_order.py]: Nice catch, I just removed it

**Review by hparfr — APPROVED** [review]

**Comment by PicchiSeba** [comment]
Is there anything I could do to fix the [pre-commit error](https://github.com/OCA/field-service/actions/runs/4043176312/jobs/6951788951) preventing this PR merge?

**Comment by max3903** [comment]
@PicchiSeba You can try downgrading node in the pre-commit config to see it fixes the issue or look at other repos in the OCA to check if they got the issue and how they fixed it.

**Comment by PicchiSeba** [comment]
included in #1039

**Comment by PicchiSeba** [comment]
closing because it has been fixed already

**Pull request closed without merging**